src/main/java/Bobby.java

```diff
@@ -196,7 +197,7 @@ private static void handle_mark() throws Exception {
      *
      * @throws Exception if there is an issue processing the next action.
      */
-    private static void handle_unmark() throws Exception {
+    private static void handleUnmark() throws Exception {
         Task curr_task;
         if (scan.hasNextInt()) {
             curr_task = taskList.retrive_task(scan.nextInt());
```

Comment:
Should the retrieve_task method be written in camel case instead to follow the coding convention?